etest/__init__.py

```diff
@@ -49,7 +46,7 @@ def echo_check_verbose(check):
 @click.option("-v", "--verbose", is_flag=True, default=False, help="provide more output")
 @click.version_option(information.VERSION)
 @click.argument("ebuilds", nargs=-1)
-def etest(dry_run, fast, jobs, quiet, verbose, ebuilds):
+def etest(dry_run: bool, fast: bool, jobs: int, quiet: bool, verbose: bool, ebuilds: Optional[Tuple[str]]) -> None:
```

Comment:
I dont know, I think you added this on #92

Reply:
We really need to figure out how to remove this commit from this review then.  What's the tree say about this branch?  If you don't spot anything obvious I'll pull and look as well.

Reply:
I have a clean local branch that I plan to upload so we can land this more cleanly.

Reply:
Hmm, can you just force push that over this pull request and see what survives?

Reply:
Im worried this will mess with comments and reviews. If you think that wont be a problem, I can still force push it.

Reply:
Yeah, it might do that.  Let's finish the current comments and do this before the next round of reviews.  Can you provide a paste of the graph for this pull request as well so we can see what happened?